apps/meteor/client/views/marketplace/AppDetailsPage/tabs/AppDetails/AppDetails.tsx

```diff
@@ -83,6 +84,14 @@ const AppDetails = ({ app }: { app: AppInfo }): ReactElement => {
 								<ExternalLink to={support} />
 							</Box>
 						</Box>
+						{documentationUrl && (
+							<Box>
+								<Box fontScale='h4' color='hint'>
+									{t('Documentation')}
+								</Box>
+								<ExternalLink to={documentationUrl} />
+							</Box>
```

Comment:
do we need a box here? there is anything that inherits bom box? maibe a div? fragment?

Reply:
Fixed